client/js/components/statement/assessmentTable/DpExportModal.vue

```diff
@@ -62,6 +62,25 @@
                 v-model="exportChoice.pdf.anonymous" />
             </fieldset>
 
+            <fieldset
+              v-if="options.pdf.newPagePerStn && view === 'original_statements'"
+              class="u-mb-0_5 u-pb-0_5">
+<!--              <legend-->
+<!--                class="sr-only"-->
+<!--                v-text="" />-->
+              <dp-checkbox
+                data-cy="exportModal:newPagePerStn"
+                id="pdfNewPagePerStn"
+                :label="{
+                  bold: true,
+                  text: Translator.trans('export.newPagePerStatement')
+                }"
+                name="newPagePerStn"
+                value="newPagePerStn"
```

Comment:
For what do we need this value?

Reply:
We don't need it, removed in acf774c3a195fe8d1d192aa8ac15a25a3e0358eb